unit_tests/web_tests/test_web_cloud.py

```diff
@@ -1186,7 +1184,7 @@ def test_web_cloud_metadata_update_priority_by_typing_string(self):
         self.assertTrue(self.page.metadata_priority_popup_exists())
         wta.assertHasNotAttribute('metadata', metadata_name, 'priority', 'invalid-web-test', group=self.gvar['base_group'], metadata_cloud=cloud_name)
 
-    @unittest.skip("Not working in production (issue 319)")
+    # @unittest.skip("Not working in production (issue 319)")
```

Comment:
delete me

Reply:
removed in [d218695](https://github.com/hep-gc/cloudscheduler/pull/432/commits/d21869592fba01146d475921fdf88924e5428334)